packages/cyberstorm/src/components/Layout/LoginLayout/BetaLoginLayout.module.css

```diff
@@ -188,6 +188,11 @@
   line-height: var(--line-height--l);
 }
 
+.loginButton > a > div > svg {
+  width: 1rem;
```

Comment:
I don't think this does what you assume it does. Change the values to 10rem and see what happens. Not 100% sure (grav might be the person to ask) but you might want to consider using the size prop of `FontAwesomeIcon`

Reply:
Sadly this does what I want. Tried to figure this out with the size prop, but that does nothing. The problems we've had with `FontAwesomeIcon` component should be figured out in a separate task. I think I made a task about it somewhere 🤔

Reply:
To reduce a change there's some miscommunication...

1. Care to elaborate what it is that you want? Based on the selector I'd assume this is to change the size of the SVGs in the link buttons. But when tested (Storybook/Chrome), having this thing or removing it altogether had zero effect on the icons or the buttons (I took screenshots before and after and they look identical).
2. Changing the to e.g. 5rem as I suggested increases the size of the OW icon a bit, but nowhere near the expected 5x. GitHub/Discord icon size doesn't change at all. However, height of all the buttons do change dramatically.
3. Getting this far in writing this question I now assume you're trying to target the OW logo and not the others. And since OW logo isn't a FontAwesomeIcon like the rest so the size prop doesn't work. Is this correct? Your answer was a bit ambiguos.
4. If the above is correct it's a bit problematic since selector chooses all three icons while you want to change just one, and the height property works in unexpected way. Adding a class to e.g. OW `<a>` to target only it would resolve the first issue, and maybe the height property could just be dropped since it doesn't affect the visible size of the icon? I don't consider this a blocker though.
5. Care to link to that task you mentioned?

Reply:
I'm not using storybook for development right now. I've only used the dev env and in there in the login page, the two FA icons are not shown at all without this, because the parent div doesn't have a size. I could have added a wrapper element for the FA icons and used a component css class in it, but seemed a bit overkill. Anyways I guess there are some differences in the reset css of dev env and storybook.

I'll link you the task in discord.

Reply:
> I'm not using storybook for development right now.

IMO, since the page has a Story, you should use Storybook too. A bad analogy would be "I didn't update the unit tests because I didn't use them for development" I guess?

> the two FA icons are not shown at all without this, because the parent div doesn't have a size

Tested master branch locally, this doesn't seem to be the case. Instead, the icons consume all the width available to them, and thus result in different heights, affecting the button dimensions. Bringing this out since it might help writing a task for this and coming up with a solution.

> I guess there are some differences in the reset css of dev env and storybook

Is there? Shouldn't they import the same stuff from `cyberstorm-styles`? If there's indeed a difference, create a task about seeing if there's a real reason for it, and remove it otherwise. I'd assume having such a difference will cause similar confusion in the future too.